cpu/Makefile.include.gnu

```diff
@@ -9,6 +9,16 @@ endif
 export AS = $(PREFIX)as
 export LINK = $(PREFIX)gcc
 export SIZE = $(PREFIX)size
+ifeq (,$(OBJCOPY))
```

Comment:
I actually would prefer the usage of `command -v $(PREFIX)objcopy gobjcopy objcopy false | head -1` as `which` is not very portable. Although I have the impression that many parts of our build system depends on `bash` explicitely, this might change in the future (?)